packages/ui-components/src/components/Button.js

```diff
@@ -56,7 +60,7 @@ Button.defaultProps = {
 };
 
 export const LightPrimaryButton = styled(Button)`
-  background-color: ${props => props.theme.palette.primary.light};
+  background-color: #dceffb;
```

Comment:
Did we wanna update the value in the `theme` to this? Or perhaps have a new key for it?

Reply:
Unfortunately I wasn't able to use a theme variable because the colour for the button is not in the palette of the design. Designers just can't stick to the rules ay 😜  I'll check with Dave to see if he wants to change the theme but for now it can probably just stay as a one off